CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+0.42.0 (2022-03-17)
+-------------------
+- Allow non-default configuration directory
+- Improved OS signal handling for graceful termination
+- Relax limit on split_file_max_chunks
+- Make tap-mysql-fastsync compatible with MySQL 8
+- Fixed failing pg-to-pg fastsync on empty tables
+- Replace `ansible` with `ansible-core`
+- Bump `tap-postgres` to `1.8.3`
+- Bump `tap-snowflake` to `3.0.0`
+- Update for compatibility with MySQL 8
+- Improve signal handling
+- Fix pg_to_sf e2e tests
```

Comment:
mysql and signal handling points are already added above.
fix e2e test isn't really worth mentioning in the changelog